src/components/NavBar/NavbarLoggedOut.jsx

```diff
@@ -20,9 +19,7 @@ const NavbarLoggedOut = () => (
   <Box sx={{ flexGrow: 1 }}>
     <AppBar position="static" color="primary">
       <Toolbar sx={{ minHeight: '64px' }}>
-        <Typography variant="h4" noWrap component="div" mr="10px" aria-label="logo">
-          PASS
-        </Typography>
+      <img src='src/assets/pass-logo.png' alt='logo' className='navbar-logo' aria-label="logo"/>
```

Comment:
see other comment, but do styling inline for now. we are trying to get rid of `styles.css` file.